src/gluonts/dataset/common.py

```diff
@@ -455,7 +592,7 @@ def __call__(self, data: DataEntry) -> DataEntry:
 
 
 def load_datasets(
-    metadata: Path, train: Path, test: Optional[Path]
+    metadata: Path, train: Path, test: Optional[Path], use_arrow=True
```

Comment:
One thing I like about gluonts in general is how few options and settings there are. What do you think about optionally specifying the file format in MetaData? Inferring could also be one option.